### ISSUE TITLE
Distributed rendering

### ISSUE THREAD

**Issue by llbit** [chunky-dev/chunky#7]
Client/server interface for distributed rendering.

**Comment by magneticflux-** [comment]
@llbit I have used [JPPF](http://www.jppf.org/) for similar Java distributed computation before and it's worked well. The main goal of it is to be able to be set up/integrated extremely quickly, while also being very powerful. The data caching feature (to prevent sending lots of render data over and over) and the node balancing features seem like they would be useful, and it even comes with an OpenCL demo, so you could submit jobs that require a GPU and jobs that require a CPU separately.

In short, it's in active development, native Java, and, from my experience, has handled massive computational loads (read: several thousand games of Super Mario Bros being played by NEAT-style neural networks). While it will take some effort to restructure Chunky to be more modular so it can be used, it's a lot better than having to write your own networking and distribution code at the same time.
